.config/nextest.toml

```diff
@@ -59,14 +55,17 @@ or test(test_stake_all)
 or test(test_fullnode_traffic_control_spam_delegated)
 '''
 
+# Custom profiles are layered on top of the default profile.
+[profile.ci]
+# Show skipped tests in the CI output.
+status-level = "skip"
+
+[profile.ci.junit]
+path = "junit.xml"
+
+# Custom profiles are layered on top of the default profile.
 [profile.simtestnightly]
-# Print out output for failing tests as soon as they fail, and also at the end
-# of the run (for easy scrollability).
-failure-output = "immediate"
 # Show skipped tests in the CI output.
 status-level = "fail"
```

Comment:
is that comment correct?
Same comment for "fail" and "skip" some lines above.

Reply:
it wasn't, I changed the value back and forth for testing and the comment didn't follow
Thanks